### ISSUE TITLE
Add DEICODE as option in analysis

### ISSUE THREAD

**Issue by adswafford** [qiita-spots/qiita#2863]
https://library.qiime2.org/plugins/deicode/19/

**Comment by antgonza** [comment]
At this stage we are only loading Q2 core plugins as this assures us having full support and compatibility as we move between Q2/plugin versions. Please coordinate with the Q2 plugin developers to verify when and if their plugin will become part of the core; once this is done, the plugin will be automatically added to Qiita.

**Comment by adswafford** [comment]
Gotcha, thanks for clarifying the process for loading q2 plugins into Qiita.

**Comment by mestaki** [comment]
Just a comment about "core-plugins". The Caporaso team is actively moving away from the idea of "core" plugins, meaning that in the not so distant future you no longer get any core plugins when you install Q2, rather all plugins needs to be downloaded individually from the Library. I believe they will provide a convenient "core" package in the new Library that will download basically what is currently considered as "core plugins" but I wouldn't count on them adding any other new core packages to that core, like q2-deicode or otherwise.
Perhaps something worth considering for future of Qiita to not rely on what Q2 considers "core"?

Sorry wasn't sure if this should be placed here or on a new issue, feel free to reclose.

**Comment by antgonza** [comment]
Yup, that make sense; we could add that "core plugin" description + any other plugins we want and add them to the qp-qiime2 plugin (Qiita's QIIME 2 plugin), as an example: https://github.com/qiita-spots/qp-qiime2/pull/63/files
